packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -412,14 +412,14 @@ var RESTAdapter =  Adapter.extend(BuildURLMixin, {
     @param {Object} query
     @return {Promise} promise
   */
-  query: function(store, type, query) {
-    var url = this.buildURL(type.modelName, null, null, 'query', query);
+  query: function(store, type, options) {
+    var url = this.buildURL(type.modelName, null, null, 'query', options.query);
```

Comment:
Ha yes I forgot to change that. I was hesitating between pushing `options.query` or `options` to keep `findQuery` working. But since `findQuery`was removed in my last rebase, I decided to use options directly. Updating now !